compiler/AST/iterator.cpp

```diff
@@ -645,8 +696,7 @@ buildHasMore(IteratorInfo* ii, BlockStmt* singleLoop) {
     // In copied expressions, replace _ic with hasMore->_this .
     map.put(ic, ii->hasMore->_this);
 
-    // NOAKES 2014/12/04 Why is this copy here?
-    testBlock = cforLoop->testBlockGet()->copy(&map);
+    testBlock = cforLoop->testBlockGet();
```

Comment:
How sure are you we don't need the copy? I don't have any opinion one way or the other, but I'd rather leave the copy with the comment in if we're not sure